float32[] frontierPredictedArea
int32[] frontierNumber
bool[] touchMapEdges
float32[] frontierPredictedAreaGreyPixel
